### PR TITLE
Cook/input forcing

### PR DESCRIPTION
**Pull request by mandresm**
See the resulting recipe here:
- https://esm-tools.readthedocs.io/en/cook-input_forcing/cookbook.html

### CODE REVIEW

File: docs/recipes/add_forcing_input.rst [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+**Feature available since version:** 4.2
+
+There are several ways of including a new forcing or input file into your experiment
+depending on the degree of control you'd like to achieve. An important clarification is
+that ``<forcing/input>_sources`` file dictionary specifies the **sources** (paths to
+the files in the pools or personal folders, that need to be copied or linked into the
+experiment folder). On the other hand ``<forcing/input>_files`` specifies which of these
+sources are to be **included in the experiment**. This allows us to have many sources
+available already for the user and then the user can simply select which of them wants
```
Comment (nwieters) [inline docs/recipes/add_forcing_input.rst]: This allows us to have many sources available already for the user and then the user can simply select which of them wants to use from ``<forcing/input>_files``.

This allows us to have many sources already available to the user, and the user can then simply choose which of them to use by chosing from ``<forcing/input>_files``.

File: docs/recipes/add_forcing_input.rst [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+to use from ``<forcing/input>_files``. ``<forcing/input>_in_work`` is used to copy the
+files into de the work folder (``<base_dir>/<exp_id>/run_<DATE>/work``) if necessary
```
Comment (nwieters) [inline docs/recipes/add_forcing_input.rst]: into de the

into the

File: docs/recipes/add_forcing_input.rst [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                      to: <last_year>
+
+2. Make sure the `key` for your path is defined in one of the config files that you are
+   using, inside of either ``forcing_files`` or ``input_files``. If it is not defined
+   anywhere you will have to include it in your runscript:
+
+   .. code-block:: yaml
+
+      <forcing/input>_files:
+      y        <key_for_your_file>: <key_for_your_source>
```
Comment (nwieters) [inline docs/recipes/add_forcing_input.rst]: Is this "y" a typo?

File: docs/recipes/add_forcing_input.rst [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+Copy the file in the work folder and/or rename it
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To copy the files from the forcing/input folders into de the work folder
```
Comment (nwieters) [inline docs/recipes/add_forcing_input.rst]: into de the

into the

File: docs/recipes/add_forcing_input.rst [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+(``<base_dir>/<exp_id>/run_<DATE>/work``) or rename them:
+
+1. Make sure your file and its source is defined somewhere (either in the config files
+   or in your runscript) in ``<forcing/input>_sources`` and ``<forcing/input>_files``
+   (see subsections :ref:`add_forcing_input:Source Path Already Defined in a Config File`
```
Comment (nwieters) [inline docs/recipes/add_forcing_input.rst]: The ref does not work in the final readthedocs page.

File: docs/recipes/add_forcing_input.rst [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+   and :ref:`add_forcing_input:Modify the Source of a File`).
```
Comment (nwieters) [inline docs/recipes/add_forcing_input.rst]: The ref does not work in the final readthedocs page.

**Comment by mandresm** [comment]
@nwieters , thanks for your revisions!

**Pull request merged**